src/__tests__/index.spec.ts

```diff
@@ -1,50 +1,44 @@
 import { shard } from '../';
-import { CiParallelVarsMock } from '../../__mocks__/ci-parallel-vars';
 
 describe('ShardyMcShardFace', () => {
-    it('throws when empty', () => {
+    beforeAll(() => {
         jest.unmock('ci-parallel-vars');
+    });
```

Comment:
I think this can now be removed? It only reverts the effects of `import { CiParallelVarsMock } from '../../__mocks__/ci-parallel-vars';` which is 🗑